docs/tutorials/mongoid-documents.txt

```diff
@@ -415,6 +415,33 @@ For cases when you do not want to have ``BSON::ObjectId`` ids, you can override
      field :_id, type: String, default: ->{ name }
    end
 
+A Note about the BSON Symbol type
+*********************************
+
+Because the BSON specification deprecated the BSON symbol type, the `bson` gem will serialize Ruby
+symbols into BSON strings when used on its own. However, in order to maintain backwards
+compatibility with older datasets, `mongo` gem overrides this behavior to serialize Ruby symbols as
+BSON symbols. This is necessary to be able to specify queries for documents which contain BSON
+symbols as fields.
+
+Because of this behavior, new applications should exclusively use the string type in place of the
```

Comment:
We should make it clearer in this paragraph that we are talking about the Model field ``type`` setting. So you can change this to "..new applications should not specify model fields with the ``Symbol`` type and rather the ``String`` type. Existing applications with ``Symbol`` model fields may convert those fields to Strings using one of the following methods: